root/var/www/html/freepbx/rest/lib/gateway/functions.inc.php

```diff
@@ -159,8 +159,20 @@ function gateway_generate_configuration_file($name,$mac = false){
         if (!empty($config['trunks_fxs'])){
             $i=0;
             foreach ($config['trunks_fxs'] as $trunk){
+                $gateway_ip = explode (".",$config['ipv4_new']);
```

Comment:
```suggestion
                $gateway_ip = explode(".",$config['ipv4_new']);
```

Reply:
Also use better naming, that let us know that you are splitting adresses by octet

Reply:
And add a comment that state what you're doing and why